UM/PackageManager.py

```diff
@@ -402,3 +404,51 @@ def getPackageLicense(self, filename: str) -> Optional[str]:
                                             file_info.filename)
                         license_string = None
         return license_string
+
+    ##  Find the package files by package_id by looking at the installed folder
+    @staticmethod
+    def getPackageFiles(package_id) -> List[Tuple[str, List[str]]]:
+        data_storage_dir = os.path.abspath(Resources.getDataStoragePath())
```

Comment:
Suspect issues if `Resources.getDataStoragePath()` returns `None`

Reply:
`Resources.getDataStoragePath()` always returns a `str`, so that's fine.